src/angularStorage/services/internalStore.js

```diff
@@ -1,8 +1,12 @@
 angular.module('angular-storage.internalStore', ['angular-storage.localStorage', 'angular-storage.sessionStorage'])
   .factory('InternalStore', function($log, $injector) {
 
-    function InternalStore(namespace, storage, delimiter) {
+    function InternalStore(namespace, storage, delimiter, useCache) {
       this.namespace = namespace || null;
+      if (typeof useCache === 'undefined' || useCache === undefined || useCache == null) {
```

Comment:
Actually, just set `this.useCache = useCache`. If undefined or null, it'll be false :).